geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -885,7 +911,9 @@ def _wms_get_features_type(self, ogc_server_id, wfs_url):
             headers.pop("Host")  # pragma nocover
 
         try:
-            response = self.get_http_cached(wfs_url, headers)
+            response = await asyncio.get_event_loop().run_in_executor(
+                None, get_http_cached, self.http_options, wfs_url, headers
+            )
```

Comment:
What does this do exactly, is it not the same as : 
```
response = await get_http_cached(self.http_options, url, headers)
```

Reply:
same...